src/protections/HomeserverUserPolicyApplication/UserRestrictionCapabilityRenderer.tsx

```diff
@@ -0,0 +1,154 @@
+// SPDX-FileCopyrightText: 2025 Gnuxie <Gnuxie@protonmail.com>
+//
+// SPDX-License-Identifier: AFL-3.0
+
+import {
+  describeCapabilityRenderer,
+  DescriptionMeta,
+  LiteralPolicyRule,
+} from "matrix-protection-suite";
+import { RendererMessageCollector } from "../../capabilities/RendererMessageCollector";
+import { UserRestrictionCapability } from "./UserRestrictionCapability";
+import { StringUserID } from "@the-draupnir-project/matrix-basic-types";
+import { isError, Result } from "@gnuxie/typescript-result";
+import { AccountRestriction } from "matrix-protection-suite-for-matrix-bot-sdk";
+import { renderFailedSingularConsequence } from "../../capabilities/CommonRenderers";
+import {
+  DeadDocumentJSX,
+  DocumentNode,
+} from "@the-draupnir-project/interface-manager";
+import { renderRuleClearText } from "../../commands/Rules";
+import { Draupnir } from "../../Draupnir";
+
+function renderRestrictionOptions(options: {
+  rule: LiteralPolicyRule | null;
+  sender: StringUserID;
+}): DocumentNode {
+  return (
+    <ul>
+      {options.rule ? (
+        <li>rule: {renderRuleClearText(options.rule)}</li>
+      ) : (
+        <fragment></fragment>
+      )}
+      <li>
+        sender: <code>{options.sender}</code>
+      </li>
+    </ul>
+  );
+}
```

Comment:
Can we get the full policy rendered here please?

![image](https://github.com/user-attachments/assets/724fbaa9-a920-4671-aec4-6590df86d1b3)

Reply:
ahh shite will have to be follow up now